core/api/src/app/accounts/create-contact.ts

```diff
@@ -0,0 +1,82 @@
+import { CouldNotFindContactFromAccountIdError } from "@/domain/contacts/errors"
+import { ContactType } from "@/domain/contacts"
+
+import { ContactsRepository } from "@/services/mongoose"
+import { checkedToContactAlias } from "@/domain/accounts"
+
+export const createContact = async ({
+  accountId,
+  handle,
+  displayName,
+  type,
+  incrementTxs = true,
+}: {
+  accountId: AccountId
+  handle: Username
+  type: ContactType
+  displayName: ContactAlias
+  incrementTxs?: boolean
+}): Promise<Contact | ApplicationError> => {
+  const contactsRepo = ContactsRepository()
```

Comment:
handle is not username where do you use checkedToHandle (app layer needs to have the type validations too